src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Form.js

```diff
@@ -96,8 +89,12 @@ class Form extends React.Component<Props> {
         }
     };
 
-    handleChange = (name: string, value: mixed) => {
-        this.props.store.change(name, value);
+    handleChange = (name: string, value: mixed, context: FieldChangeContext) => {
+        if (context.isDefaultValue) {
+            this.props.store.set(name, value);
```

Comment:
Maybe we could in future migrate `set` and `change` into one method and give the `context` into that methods. In that case maybe `isDefaultValue` could be misleading and maybe would need then be `noDirty: true`.